applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -478,10 +478,105 @@ static int cmd_input(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
-SHELL_STATIC_SUBCMD_SET_CREATE(hw_codec_cmd,
-			       SHELL_COND_CMD(CONFIG_SHELL, input, NULL,
-					      " Select input\n\t0: LINE_IN\n\t\t1: PDM_MIC",
-					      cmd_input),
-			       SHELL_SUBCMD_SET_END);
+static int out1l_clear(void)
+{
+	int ret;
+
+	ret = cs47l63_write_reg(&cs47l63_driver, CS47L63_OUT1L_INPUT1, 0x808000);
+	if (ret) {
+		return ret;
+	}
+
+	ret = cs47l63_write_reg(&cs47l63_driver, CS47L63_OUT1L_INPUT2, 0x808000);
+	if (ret) {
+		return ret;
+	}
+
+	return 0;
+}
+
+/**
+ * @brief Shell command to choose between left, right, and mixed stereo as the I2S output
+ *
+ * @param shell Shell instance
+ * @param argc Number of arguments
+ * @param argv Array of arguments
+ *
+ * @return 0 on success, -EINVAL on error
+ */
+static int cmd_i2s_output(const struct shell *shell, size_t argc, char **argv)
```

Comment:
```suggestion
static int cmd_i2s_to_output_mapping(const struct shell *shell, size_t argc, char **argv)
```

Reply:
I probably won't change the command itself though